<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetreChargement</class>
 <widget class="QDialog" name="fenetreChargement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Business - Chargement</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/logo.png</normaloff>../ressources/logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>361</width>
     <height>154</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formulaireLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="titreCharger">
      <property name="text">
       <string>Choisissez une sauvegarde</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="cheminLabel">
      <property name="text">
       <string>Chemin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="cheminLayout">
      <item>
       <widget class="QLineEdit" name="cheminLineEdit">
        <property name="placeholderText">
         <string>C:/Users/Bill/Documents/Business/MonRegistre.bss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cheminPushButton">
        <property name="text">
         <string>RÃ©pertoire...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="boutonsLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="chargerPushButton">
        <property name="text">
         <string>Charger</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="passerPushButton">
        <property name="text">
         <string>Passer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
